meson.build

```diff
@@ -120,6 +120,13 @@ dxvk_version = vcs_tag(
   input:  'version.h.in',
   output: 'version.h')
 
+# Common kwargs for targets
+common_opts = { 'override_options': ['cpp_std=' + dxvk_cpp_std], 'gnu_symbol_visibility': 'inlineshidden' }
```

Comment:
Where does the gnu_symbol_visibility thingy come from? We're not using it right now and this doesn't look like something that would work with MSVC.

Reply:
Added this as a bonus (meson ~v0.48). I guess it should not have any effect on MSVC, same as `vs_module_defs` have no effect on GCC and enabled unconditionally. Instead will be used by builds like winelib or dxvk native and kinds.

Also `'path' / 'to' / 'file'` concatenation was used as one of new features, and `objects` replaced with more (i guess) correct version.

Reply:
You mean separate commit in this PR, or separate PR? Or I should simply remove this and forget?